Dockerfile

```diff
@@ -19,20 +20,14 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update -y  && \
         apt-get clean -y && \
         rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-# instead of virtual env, just use python3.7 as default
+# instead of virtual env, just use python3 as default
 RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1
 
 # Install Tini
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
 RUN chmod +x /tini
 
-# needed for some system package
-# TODO: investigate
-RUN ln -s /bin/tar/ /bin/gtar
-
-RUN pip3 install wheel && \
-    pip3 install civis-jupyter-notebook==${CIVIS_JUPYTER_NOTEBOOK_VERSION} \
-      cbor2==4.1.2 && \
```

Comment:
This `cbor2` dependency seems unnecessary now.  The `wheel` dependency is not necessary with the latest versions of python and pip.